release notes/v0.38.3.md

```diff
@@ -0,0 +1,9 @@
+k6 v0.38.3 is a patch release containing a single fix
+
+### Threshold over already defined sub-metrics will result in an error ([#2538](https://github.com/grafana/k6/issues/2538))
+
+There was a bug where we were checking if no already added submetrics is added again. Unfortunately we didn't check that this will work with the one submetric we have by default `http_req_duration{expected_response:true}`. After v0.38.0 defining a threshold on it would result in an error.
```

Comment:
```suggestion
There was a bug where we were checking if a submetric had already been added. Unfortunately, we didn't check that this will work with the one submetric we have by default `http_req_duration{expected_response:true}`. After v0.38.0 defining a threshold on it would result in an error.
```

Maybe something like this?